doc/user/content/transform-data/patterns/top-k.md

```diff
@@ -57,45 +58,53 @@ INSERT INTO cities VALUES
 To fetch the three most populous cities in each state:
 
 ```mzsql
-SELECT state, name FROM
+SELECT state, city FROM
     (SELECT DISTINCT state FROM cities) grp,
     LATERAL (
-        SELECT name FROM cities
+        SELECT city, pop FROM cities
         WHERE state = grp.state
         ORDER BY pop DESC LIMIT 3
-    );
-```
-```nofmt
-AZ  Phoenix
-CA  Los_Angeles
-CA  San_Diego
-CA  San_Jose
-IL  Chicago
-NY  New_York
-TX  Houston
-TX  San_Antonio
-TX  Dallas
+    )
+ORDER BY state, pop DESC;
 ```
 
+The query returns the following results:
+
+| state | city        |
+| ----- | ----------- |
+| AZ    | Phoenix     |
+| CA    | Los_Angeles |
+| CA    | San_Diego   |
+| CA    | San_Jose    |
+| IL    | Chicago     |
+| NY    | New_York    |
+| TX    | Houston     |
+| TX    | San_Antonio |
+| TX    | Dallas      |
+
 Despite the verbosity of the above query, Materialize produces a straightforward
 plan:
 
 ```mzsql
-EXPLAIN SELECT state, name FROM ...
+EXPLAIN SELECT state, city FROM ...
 ```
+
+The explain returns the following:
+
 ```nofmt
 Explained Query:
-  Project (#1, #0)
-    TopK group_by=[#1] order_by=[#2 desc nulls_first] limit=3
-      ReadStorage materialize.public.cities
+  Finish order_by=[#0{state} asc nulls_last, #2{pop} desc nulls_first] output=[#0..=#2]
+    TopK group_by=[#0{state}] order_by=[#2{pop} desc nulls_first] limit=3 // { arity: 3 }
+      Project (#1{city}, #0{state}, #2{pop}) // { arity: 3 }
+        ReadStorage materialize.public.cities // { arity: 3 }
```

Comment:
I see the TopK group_by in the explain plan, so I'm thinking that's good.

W/o the order by, the explain is:
```
Explained Query:
  Project (#1{city}, #0{state}) // { arity: 2 }
    TopK group_by=[#1{state}] order_by=[#2{pop} desc nulls_first] limit=3 // { arity: 3 }
      ReadStorage materialize.quickstart2.cities // { arity: 3 }
```
So ... I'm hoping the diff doesn't indicate a significant perf hit.